ci/jenkins/Jenkins_steps.groovy

```diff
@@ -665,6 +679,41 @@ def test_unix_python3_mkldnn_cpu() {
     }]
 }
 
+def test_unix_python2_mkldnn_mkl_cpu() {
+    return ['Python2: MKLDNN-MKL-CPU': {
+      node(NODE_LINUX_CPU) {
+        ws('workspace/ut-python2-mkldnn-mkl-cpu') {
+          try {
+            utils.unpack_and_init('mkldnn_mkl_cpu', mx_mkldnn_lib, true)
+            python2_ut('ubuntu_cpu')
+            utils.publish_test_coverage()
+          } finally {
+            utils.collect_test_results_unix('nosetests_unittest.xml', 'nosetests_python2_mkldnn_cpu_unittest.xml')
+            utils.collect_test_results_unix('nosetests_train.xml', 'nosetests_python2_mkldnn_cpu_train.xml')
+            utils.collect_test_results_unix('nosetests_quantization.xml', 'nosetests_python2_mkldnn_cpu_quantization.xml')
+          }
+        }
+      }
+    }]
+}
+
+def test_unix_python3_mkldnn_mkl_cpu() {
```

Comment:
I don't think we need python 2 and 3

Reply:
I find it will redownload and install mkl in unittest Python2: MKLDNN-MKL-CPU http://jenkins.mxnet-ci.amazon-ml.com/blue/rest/organizations/jenkins/pipelines/mxnet-validation/pipelines/unix-cpu/branches/PR-13607/runs/3/nodes/193/steps/479/log/?start=0

Reply:
Yeah that's expected. The Docker cache will only activate after this PR has been merged into master. We do that to avoid growing the cache too heavily and instead only use the master state for caching.

In your case, ubuntu_mkl has been added by you and thus the cache diverges and a non-cached version gets used at that point. It works as intended :)